ios/src/platform_ios.mm

```diff
@@ -54,22 +54,33 @@ bool isContinuousRendering() {
 
 NSString* resolvePath(const char* _path) {
 
-    NSString* path = [NSString stringWithUTF8String:_path];
+    NSString* pathString = [NSString stringWithUTF8String:_path];
+
+    NSURL* resourceFolderUrl = [NSURL fileURLWithPath:[[NSBundle mainBundle] resourcePath]
```

Comment:
As for requiring Resources, this is fine for a start. Eventually we're going to need to deal with Asset Catalog usage which is not stored in Resources, but that's farther down the path.